engine/src/main/java/org/terasology/rendering/cameras/Camera.java

```diff
@@ -204,6 +206,22 @@ public Vector3f getViewingDirection() {
         return viewingDirection;
     }
 
+    /**
+     * Get the gaze direction.
+     * @return the gaze direction, a quaternion.
+     */
+    public Quat4f getGazeDirection() {
```

Comment:
Honestly, I just needed to have this process:
LocalPlayerSystem: Hey camera, please look at X.
VR camera: Can't do that buddy, my camera only moves where my player moves his head.
LocalPlayerSystem: Alright camera, where are we looking?
VR camera: Here's the player's head pose.

vs.

LocalPlayerSystem: Look here, camera.
Normal Camera: Okay.
LocalPlayerSystem: Where are we looking?
Normal Camera: (gives the direction that was given earlier).

I agree that the viewing direction is probably redundant here, and in fact, there are no references elsewhere in the code, so it's probably fine to merge the two. When I initially decided not to use `getViewingDirection()` to store that info, I was just worried that I would break something else that depended on it working as it did earlier.